### PR TITLE
3196 update plasma geometry and shaping documentation to include new models

### PR DESCRIPTION
**Pull request by chris-ashe**
## Checklist

I confirm that I have completed the following checks:

- [ ] I have justified any large differences in the regression tests caused by this pull request in the comments.
- [ ] I have added new tests where appropriate for the changes I have made.
- [ ] If I have had to change any existing unit or integration tests, I have justified this change in the pull request comments.
- [ ] If I have made documentation changes, I have checked they render correctly.
- [ ] I have added documentation for my change, if appropriate.

### CODE REVIEW

File: documentation/proc-pages/images/plasma_geometry.png
Comment (timothy-nunn) [inline documentation/proc-pages/images/plasma_geometry.png:1]: Could you put an arrow for $a$?

File: documentation/proc-pages/physics-models/plasma_geometry.md [hunk 1]
```diff
@@ -1,105 +1,682 @@
...
-- `ishape = 8` -- the input values for `kappa` and `triang` are used directly and the 95% flux 
+---------------------------------------------------------------------
+- `ishape = 8` -- The input values for `kappa` and `triang` are used directly and the 95% flux 
   surface values are calculated using the FIESTA fit from `ishape = 7`.
+---------------------------------------------------------------------
+- `ishape = 9` -- The input values for `triang` and `rli` are used, `kappa` and the 95% flux 
+  surface values are calculated.
+
+  $$
+   \kappa = \left(\left(1.09+\frac{0.26}{l_i}\right)\left(\frac{1.5}{A}\right)^{0.4}\right)
```
Comment (timothy-nunn) [inline documentation/proc-pages/physics-models/plasma_geometry.md:122]: Its not clear if this also comes from reference [1]?

File: documentation/proc-pages/images/plasma_inboard_surface_area.png
Comment (timothy-nunn) [inline documentation/proc-pages/images/plasma_inboard_surface_area.png:1]: $x = x_i$?

File: documentation/proc-pages/images/plasma_outboard_surface_area.png
Comment (timothy-nunn) [inline documentation/proc-pages/images/plasma_outboard_surface_area.png:1]: $x=x_o$?

File: documentation/proc-pages/images/plasma_inboard_surface_area.png
Comment (timothy-nunn) [reply documentation/proc-pages/images/plasma_inboard_surface_area.png:1]: And the same for $\theta$

File: documentation/proc-pages/images/plasma_outboard_surface_area.png
Comment (timothy-nunn) [reply documentation/proc-pages/images/plasma_outboard_surface_area.png:1]: And the same for $\theta$

File: documentation/proc-pages/physics-models/plasma_geometry.md [hunk 2]
```diff
@@ -1,105 +1,682 @@
...
+| `triang`, $\delta$      | Plasma separatrix triangularity      |
+
+| Output Variable | Description                          |
+|----------|--------------------------------------|
+| `perim`       | Plasma poloidal perimeter length (m) |
+
+Inboard arc
+
+$$
+\mathtt{denomi} = \frac{(\delta^2 + \kappa^2 - 1.0)}{(2.0 \times (1.0 - \delta)) + \delta}
```
Comment (j-a-foster) [inline documentation/proc-pages/physics-models/plasma_geometry.md:531]: I think the `+\delta` at the end of the denominator is supposed to be outside the fraction?

File: documentation/proc-pages/physics-models/plasma_geometry.md [hunk 3]
```diff
@@ -1,105 +1,682 @@
...
+$$
+
+$$
+\mathtt{xli} = a \times (\mathtt{denomi} + 1.0 - \delta)
+$$
+
+Outboard arc
+
+$$
+\mathtt{denomo} = \frac{(\delta^2 + \kappa^2 - 1.0)}{(2.0 \times (1.0 + \delta)) - \delta} \\
```
Comment (j-a-foster) [inline documentation/proc-pages/physics-models/plasma_geometry.md:545]: Same thing as above comment with the `-\delta` at the end of the denominator

File: documentation/proc-pages/physics-models/plasma_geometry.md [hunk 4]
```diff
@@ -1,105 +1,682 @@
...
+| `rminor`, $a$        | Plasma minor radius (m)              |
+| `kappa`, $\kappa$      | Plasma separatrix elongation         |
+| `triang`, $\delta$      | Plasma separatrix triangularity      |
+
+| Output Variable | Description                          |
+|----------|--------------------------------------|
+| `xsect0`       | Plasma cross-sectional area (m^2) |
+
+$$
+\mathtt{denomi} = \frac{(\delta^2 + \kappa^2 - 1.0)}{(2.0 \times (1.0 - \delta)) + \delta}
```
Comment (j-a-foster) [inline documentation/proc-pages/physics-models/plasma_geometry.md:628]: Same issue with `+\delta` at the end of the denominator as above

File: documentation/proc-pages/physics-models/plasma_geometry.md [hunk 5]
```diff
@@ -1,105 +1,682 @@
...
+$$
+\mathtt{cti} = \cos(\mathtt{thetai})
+$$
+
+$$
+\mathtt{sti} = \sin(\mathtt{thetai})
+$$
+
+$$
+\mathtt{denomo} = \frac{(\delta^2 + \kappa^2 - 1.0)}{(2.0 \times (1.0 + \delta)) - \delta}
```
Comment (j-a-foster) [inline documentation/proc-pages/physics-models/plasma_geometry.md:648]: And again

**Review by j-a-foster — CHANGES_REQUESTED** [review]
Docs generally look good, just need to put units in math environment in tables and address comments.

File: documentation/proc-pages/physics-models/plasma_geometry.md (hunk 1 above)
Comment (chris-ashe) [reply documentation/proc-pages/physics-models/plasma_geometry.md:122]: Only the 95% values are. Which is referenced with [1]

File: documentation/proc-pages/physics-models/plasma_geometry.md [hunk 6]
```diff
@@ -1,105 +1,682 @@
...
+| `kappa`, $\kappa$      | Plasma separatrix elongation         |
+| `triang`, $\delta$      | Plasma separatrix triangularity      |
+
+
+| Output Variable | Description                          |
+|----------|--------------------------------------|
+| `pperim`       | Plasma Poloidal perimeter length [m] |
+| `sarea`       | Plasma surface area [m^2] |
+| `xarea`       | Plasma cross-sectional area [m^2] |
+| `vol`       | Plasma volume [m^3] |
```
[the remark this answers is not in the document]
Comment (chris-ashe) [reply documentation/proc-pages/physics-models/plasma_geometry.md:333]: Ammended

**Review by j-a-foster — APPROVED** [review]
Happy with the corrections wrt my previous comments.

**Review by timothy-nunn — APPROVED** [review]

**Pull request merged**